ffbs-parker-nodeconfig/Makefile

```diff
@@ -8,11 +8,11 @@ PKG_LICENSE:=MIT
 
 PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
 
-include $(INCLUDE_DIR)/package.mk
+include $(TOPDIR)/../package/gluon.mk
 
 define Package/ffbs-parker-nodeconfig
   TITLE:=Automatic node configuration for parker networks
-  DEPENDS:=+kmod-wireguard +wireguard-tools +luaposix +usign +gluon-radv-filterd +gluon-ebtables-filter-ra-dhcp
+  DEPENDS:=+gluon-core +libgluonutil +gluon-mesh-vpn-core +kmod-wireguard +wireguard-tools +luaposix +usign +gluon-radv-filterd +gluon-ebtables-filter-ra-dhcp
 endef
 
 define Package/ffbs-parker-nodeconfig/description
```

Comment:
Unrelated change to VPN provider? Please split commit or comment.

Reply:
So this change is part of `add parker provider to ffbs-parker-nodeconfig`.
This ties it a little further towards gluon and also requires to have a gluon vpn available.
Therefore I think it is useful to have this change together.

Without the change to gluon.mk, the `check_site.lua` would not be executed.
Without `gluon-mesh-vpn-core` the additional `gluon/mesh-vpn/provider` file does not make much sense.
What do you think?